coral-base-list/src/scripts/BaseList.js

```diff
@@ -196,9 +196,10 @@ const BaseList = (superClass) => class extends superClass {
   /** @ignore */
   focus() {
     if (!this.contains(document.activeElement)) {
-      const items = this._getSelectableItems();
+      const items = this._getSelectableItems().filter(item => !item.hasAttribute('hidden'));
       if (items.length > 0) {
-        items[0].focus();
+        const firstFocusable = items.find(item => item.tabIndex === 0);
+        (firstFocusable || items[0]).focus();
```

Comment:
Why do we have this check item.tabIndex === 0 ?
What if the first item is button or anchorlink which are focusable but don't require tabindex=0?

Shouldn't we fix the same at _focusFirstItem and _focusLastItem etc.. which are called on key:home and key:end?

Reply:
We want to prioritize the active or first selected item over the first item.

Reply:
We are identifying firstFocusable by "item.tabIndex === 0" but items like buttons and anchorlinks won't have tabindex=0, they are by default focusable.
Lets say there are list items in following order . 
button,
coral-seelct-list
coral-seelct-list

AFAIK here firstFocusable should be button, but button won;'t have tabindex=0. 
So we will be focusing on second item 'coral-seelct-list' instead of button.

Please let me know if my understanding above isn't correct?